src/__tests__/cache.test.mjs

```diff
@@ -87,7 +87,7 @@ test('observing cache changes', async (t) => {
   t.snapshot(events);
 });
 
-test('reading cache', async (t) => {
+test.skip('reading cache', async (t) => {
```

Comment:
Meant to revisit before merging? Or intentionally skipped?

Reply:
It's a todo item.

Needs a proper `.zip` archive to work.